pkg/cache/clusterqueue_test.go

```diff
@@ -98,206 +98,238 @@ func TestFitInCohort(t *testing.T) {
+		usage              resources.FlavorResourceQuantitiesFlat
+		clusterQueue       []*kueue.ClusterQueue
 		enableLendingLimit bool
 	}{
 		"full cohort, empty request": {
 			request: resources.FlavorResourceQuantitiesFlat{},
 			wantFit: true,
-			cq: &ClusterQueueSnapshot{
-				Name: "CQ",
-				Cohort: &CohortSnapshot{
-					Name: "C",
-					RequestableResources: resources.FlavorResourceQuantitiesFlat{
-						{Flavor: "f1", Resource: corev1.ResourceCPU}:    5,
-						{Flavor: "f1", Resource: corev1.ResourceMemory}: 5,
-						{Flavor: "f2", Resource: corev1.ResourceCPU}:    5,
-						{Flavor: "f2", Resource: corev1.ResourceMemory}: 5,
-					}.Unflatten(),
-					Usage: resources.FlavorResourceQuantitiesFlat{
-						{Flavor: "f1", Resource: corev1.ResourceCPU}:    5,
-						{Flavor: "f1", Resource: corev1.ResourceMemory}: 5,
-						{Flavor: "f2", Resource: corev1.ResourceCPU}:    5,
-						{Flavor: "f2", Resource: corev1.ResourceMemory}: 5,
-					}.Unflatten(),
-				},
-				ResourceGroups: nil,
+			usage: resources.FlavorResourceQuantitiesFlat{
+				{Flavor: "f1", Resource: corev1.ResourceCPU}:    5_000,
+				{Flavor: "f1", Resource: corev1.ResourceMemory}: 5,
+				{Flavor: "f2", Resource: corev1.ResourceCPU}:    5_000,
+				{Flavor: "f2", Resource: corev1.ResourceMemory}: 5,
+			},
+			clusterQueue: []*kueue.ClusterQueue{
+				utiltesting.
+					MakeClusterQueue("CQ").
+					ResourceGroup(
+						*utiltesting.MakeFlavorQuotas("f1").
+							Resource(corev1.ResourceCPU, "5").
+							Resource(corev1.ResourceMemory, "5").
+							Obj(),
+						*utiltesting.MakeFlavorQuotas("f2").
+							Resource(corev1.ResourceCPU, "5").
+							Resource(corev1.ResourceMemory, "5").
+							Obj(),
+					).
+					Cohort("C").
+					Obj(),
 			},
 		},
 		"can fit": {
 			request: resources.FlavorResourceQuantitiesFlat{
-				{Flavor: "f2", Resource: corev1.ResourceCPU}:    1,
+				{Flavor: "f2", Resource: corev1.ResourceCPU}:    1_000,
 				{Flavor: "f2", Resource: corev1.ResourceMemory}: 1,
 			},
 			wantFit: true,
-			cq: &ClusterQueueSnapshot{
-				Name: "CQ",
-				Cohort: &CohortSnapshot{
-					Name: "C",
-					RequestableResources: resources.FlavorResourceQuantitiesFlat{
-						{Flavor: "f1", Resource: corev1.ResourceCPU}:    5,
-						{Flavor: "f1", Resource: corev1.ResourceMemory}: 5,
-						{Flavor: "f2", Resource: corev1.ResourceCPU}:    5,
-						{Flavor: "f2", Resource: corev1.ResourceMemory}: 5,
-					}.Unflatten(),
-					Usage: resources.FlavorResourceQuantitiesFlat{
-						{Flavor: "f1", Resource: corev1.ResourceCPU}:    5,
-						{Flavor: "f1", Resource: corev1.ResourceMemory}: 5,
-						{Flavor: "f2", Resource: corev1.ResourceCPU}:    4,
-						{Flavor: "f2", Resource: corev1.ResourceMemory}: 4,
-					}.Unflatten(),
-				},
-				ResourceGroups: nil,
+			usage: resources.FlavorResourceQuantitiesFlat{
+				{Flavor: "f1", Resource: corev1.ResourceCPU}:    5_000,
+				{Flavor: "f1", Resource: corev1.ResourceMemory}: 1,
```

Comment:
(f1, memory)'s number changed from the original test. please revert

Reply:
fix: https://github.com/kubernetes-sigs/kueue/pull/2655/commits/21c2d06d49d0db8ae4331b11548040548c84f29b